python/test/pyJointAxis_TEST.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
```

Comment:
This is not used. Do we need to add a `deepcopy` test?

Reply:
https://github.com/ignitionrobotics/sdformat/pull/960/commits/7b09b64bab3e3d483a328984ff1fcb1c237b9d9b